netbox/netbox/settings.py

```diff
@@ -859,6 +859,14 @@ def _setting(name, default=None):
         f"{plugin_name}.{queue}": RQ_PARAMS for queue in plugin_config.queues
     })
 
+    events_pipeline = plugin_config.events_pipeline
+    if events_pipeline and type(events_pipeline) in (list, tuple):
```

Comment:
An ImproperlyConfigured exception should be raised if `plugin_config.events_pipeline` is not an iterable.